stacktrace-disclosure.cs

```diff
@@ -0,0 +1,12 @@
+public void Configure(IApplicationBuilder app, IWebHostEnvironment env)  
```

Comment:
expecting a `ruleid` comment at line 5 here 👍 would also be great to have a case where this shouldn't match!